tools/codegen/api/python.py

```diff
@@ -753,12 +736,13 @@ def signature(f: NativeFunction, *, method: bool = False, pyi: bool = False) ->
     )
 
 # TODO blowtorch
+# note: removing this will be BC-breaking. A quick test shows that
+# randperm will otherwise default its dtype to torch.float64
```

Comment:
See the note. I tried removing, but it causes some tests to fail, which is why I noticed. We could either leave this in here or provide a deprecation warning and eventually take it out

Reply:
This isn't actually pyi specific, right? This controls how we actually generate bindings

Reply:
yeah that's right, Jiakai added this when he migrated python bindings. pyi doesn't actually touch it (another reason to not include in this PR, since I'm trying to only remove pya-specific hacks)